provider/datagen/src/bin/datagen.rs

```diff
@@ -244,49 +239,44 @@ fn main() -> eyre::Result<()> {
         eyre::bail!("No keys selected");
     }
 
+    let cldr_locale = matches
```

Comment:
Nit: `cldr_locale_type` or something, because it's not a locale

Reply:
renaming in follow up